canonical-websites/www.ubuntu.com
# Fix typo in newsletter thank you redirect

## Done

- s/newletter/newsletter

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that /cloud/newsletter-thank-you redirects to /openstack/newsletter-thank-you
- See that the form now sends you directly to https://www.ubuntu.com/openstack/newsletter-thank-you
